contracts/Vault.vy

```diff
@@ -1637,6 +1645,13 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
         self.erc20_safe_transferFrom(self.token.address, msg.sender, self, totalAvail - credit)
     # else, don't do anything because it is balanced
 
+    # Update cached value of delegated assets
+    #   (used to properly account for mgmt fee in `_assessFees`)
+    self.delegatedAssets -= self._strategy_delegatedAssets[msg.sender]
+    delegatedAssets: uint256 = Strategy(msg.sender).delegatedAssets()
+    self.delegatedAssets += delegatedAssets
```

Comment:
thinking if we should Min here between `delegatedAssets` and `totalAssets` from strategy, in case of a faulty strategy calc vault can still keep working with less fees

Reply:
Agreed, let's do this